src/main/kotlin/no/nav/familie/ba/sak/kjerne/simulering/SimuleringUtil.kt

```diff
@@ -105,7 +110,16 @@ fun hentTidligereUtbetaltIPeriode(periode: List<ØkonomiSimuleringPostering>): B
         (postering.posteringType == PosteringType.YTELSE && postering.beløp < BigDecimal.ZERO)
     }.sumOf { it.beløp }
     val feilutbetaling = hentFeilbetalingIPeriode(periode)
-    return if (feilutbetaling < BigDecimal.ZERO) -(sumNegativeYtelser - feilutbetaling) else -sumNegativeYtelser
+    return if (feilutbetaling < BigDecimal.ZERO) -(sumNegativeYtelser - feilutbetaling) else -sumNegativeYtelser - hentManuellePosteringerIPeriode(
+        periode
+    )
+}
+
+fun hentManuellePosteringerIPeriode(periode: List<ØkonomiSimuleringPostering>): BigDecimal {
+    val sumManuellePosteringer = periode.filter { postering ->
+        (postering.posteringType == PosteringType.YTELSE && postering.fagOmrådeKode == FagOmrådeKode.BARNETRYGD_INFOTRYGD_MANUELT && postering.beløp > BigDecimal.ZERO)
+    }.sumOf { it.beløp }
+    return if (sumManuellePosteringer > BigDecimal.ZERO) sumManuellePosteringer else BigDecimal.ZERO
```

Comment:
Trenger vi å ta hensyn til om det er pluss eller minus her? I vårt case var det betalt ut formye etter valutajusteringen, men hva om vi kommer fram til at det er utbetalt for lite? Vil det da være negative ytelsesposteringer fra `BARNETRYGD_INFOTRYGD_MANUELT`
```suggestion
    return periode.filter { postering ->
        (postering.posteringType == PosteringType.YTELSE && postering.fagOmrådeKode == FagOmrådeKode.BARNETRYGD_INFOTRYGD_MANUELT)
    }.sumOf { it.beløp }
```

Reply:
Dersom vi antar at vi bare får positive ytelser fra `BARNETRYGD_INFOTRYGD_MANUELT` burde vi da kaste en feil om vi får inn noen negative?

Reply:
FIksa i [Ha med både positive og negative manuelle posteringer](https://github.com/navikt/familie-ba-sak/pull/3195/commits/ad7ef5c39adf804e8c412ab9adf16a61f38b2cd5)